test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -593,10 +615,15 @@ describe('Filters widget', () => {
             cy.get('.modal').within(() => verifyRulesForm(testRules));
         }
 
-        interface ExtendedFilterRule extends FilterRule {
+        interface NonDynamicDropdownFilterRules {
+            useNonDynamicDropdown?: boolean;
+        }
```

Comment:
The name of the interface is a bit misleading as it contains `useNonDynamicDropdown`, which can be set to `false` and then the interface is pointing to actually `DynamicDropdownFilterRule` 🙂 

What do you think about something like:
```suggestion
        interface FilterRuleValueInputOptions {
            useStaticDropdown?: boolean;
        }
```
? It can also be named shorter I guess - `ValueInputOptions`.

Reply:
I was not sure about the naming, so thanks a lot for that comment
I like the suggestion, even the longer one (because it's being specific about the input purpose), so I'm applying it right away 🚀

Reply:
Necessary modifications has been applied 🎉